CHANGES.md

```diff
@@ -256,9 +258,11 @@ To be released.
 [#1274]: https://github.com/planetarium/libplanet/pull/1274
 [#1275]: https://github.com/planetarium/libplanet/pull/1275
 [#1278]: https://github.com/planetarium/libplanet/pull/1278
+[#1284]: https://github.com/planetarium/libplanet/issues/1284
+[#1285]: https://github.com/planetarium/libplanet/issues/1285
+[#1298]: https://github.com/planetarium/libplanet/pull/1298
 [#1289]: https://github.com/planetarium/libplanet/pull/1289
 
-
```

Comment:
The interval between each versions seems two new lines. 🤔 Was this determined as the rule? cc @planetarium/libplanet

Reply:
reverted. thanks!